api_spec.yaml

```diff
@@ -3290,6 +3485,14 @@ components:
       schema:
         type: string
 
```

Comment:
add an API to create a new tenant. In that API take in all the login methods that that tenant should have. Right now, we just have one login method for multi tenancy, so we should just have `thirdPartyEnabled?: boolean` (by default, this will be true by default).

This API should also allow to update tenant login method.

This means, by default in the core, we have to create defaultTenantId with all login methods enabled.

This API should also make sure that all login methods cannot be disabled.

Reply:
This means that all these API should return `TENANT_DOES_NOT_EXIST_ERROR` as well.

Reply:
The GET APIs that exist right now, they should return `thirdPartyEnabled` boolean as well along with status: "OK", and also other statuses

Reply:
GET API is removed

Reply:
Done